src/Console/PublishCommand.php

```diff
@@ -11,14 +11,15 @@ class PublishCommand extends Command
      *
      * @var string
      */
-    protected $signature = 'sail:publish';
+    protected $signature = 'sail:publish
+                {--force : The services that should be included in the installation}';
 
     /**
      * The console command description.
      *
      * @var string
      */
-    protected $description = 'Publish the Laravel Sail Docker files';
+    protected $description = 'Publish the Laravel Sail Docker files. If you want overwrite the existing files, you can add the `--force` option';
```

Comment:
No need for the extra explanation here.
```suggestion
    protected $description = 'Publish the Laravel Sail Docker files.';
```

Reply:
Hi @driesvints,

thanks for your suggestions.
Just for learning: is this really extra explanation? For me it was useful, because it's shown when php artisan is called.